gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/GradleUtil.java

```diff
@@ -205,4 +206,10 @@ static boolean canBeResolved(Configuration configuration) {
             Function.identity(),
             (old, rep) -> rep));
   }
+
+  private static boolean isJavaArtifact(File artifact) {
+    return artifact.getName().endsWith("jar") ||
+        artifact.getName().endsWith("war") ||
+        artifact.getName().endsWith("ear");
```

Comment:
This is the part that worries me most.
I tried to find a proper way to identify the artifacts in the configurations, but couldn't.

Anyway, the expression to identify the extension won't work with some edge cases. Uppercased files, random extensions `hear`, `rear`....

I suggest changing with sth like:
```suggestion
    return artifact.getName().toLowerCase(Locale.ROOT).matches(".+?\\.(jar|war|ear)");
```